app/lib/package/models.dart

```diff
@@ -81,6 +81,11 @@ class Package extends db.ExpandoModel<String> {
   @db.BoolProperty(required: true)
   bool isDiscontinued;
 
+  /// The package that should be used instead of the current package.
+  /// May have a value only if [isDiscontinued] is set.
+  @db.StringProperty()
+  String replacedBy;
```

Comment:
Can we augment integrity check to ensure that the package name mentioned here exists?

And that it is `null` whenever `discontinued = false`.